modules/contact-form/js/gutenblock.jsx

```diff
@@ -0,0 +1,630 @@
+	constructor( ...args ) {
+		super( ...args );
+		this.onChangeOption = this.onChangeOption.bind( this );
+	}
+
+	onChangeOption( key = null, option = null ) {
+		const newOptions = this.props.options.slice( 0 );
+		if ( 'object' === typeof key ) {
+			newOptions.push( '' );
+		} else if ( null === option ) {
+			newOptions.splice( key, 1 );
+		} else {
+			newOptions.splice( key, 1, option );
+		}
+		this.props.setAttributes( { options: newOptions } );
+	}
+
+	render() {
+		return (
+			<Fragment>
+				<GrunionFieldSettings
+					required={ this.props.required }
+					setAttributes={ this.props.setAttributes }
+				/>
+				<div className="grunion-field">
+					<GrunionFieldLabel
+						required={ this.props.required }
+						label={ this.props.label }
+						setAttributes={ this.props.setAttributes }
+					/>
+					<ol>
+						{ this.props.options.map( ( option, index )=>(
+							<GrunionOption
+								key={ index }
+								option={ option }
+								index={ index }
+								onChangeOption={ this.onChangeOption }
+							/>
+						) ) }
+					</ol>
+					<IconButton
+						icon="insert"
+						label={ __( 'Insert option' ) }
+						onClick={ this.onChangeOption }
+					> { __( 'Add' ) }</IconButton>
+				</div>
+			</Fragment>
+		)
+	}
+}
+
+class GrunionOption extends Component {
+	constructor( ...args ) {
+		super( ...args );
+		this.onChangeOption = this.onChangeOption.bind( this );
+		this.onDeleteOption = this.onDeleteOption.bind( this );
+	}
+
+	onChangeOption( x ) {
+		this.props.onChangeOption( this.props.index, x.target.value );
+	}
+
+	onDeleteOption() {
+		this.props.onChangeOption( this.props.index );
+	}
+
+	render() {
+		return (
+			<li>
+				<input
+					type="text"
+					className="option"
+					value={ this.props.option }
+					placeholder={ __( 'Enter your option value here…' ) }
+					onChange={ this.onChangeOption }
+				/>
+				<IconButton
+					icon="no"
+					label={ __( 'Remove option' ) }
+					onClick={ this.onDeleteOption }
+				/>
+			</li>
+		);
+	}
+}
+
+/**
+ * Block Registrations:
+ */
+
+registerBlockType( 'grunion/form', {
+	title: __( 'Contact Form', 'jetpack' ),
+	icon: 'feedback',
+	category: 'widgets',
+	supports: {
+		html: false
+	},
+	/* // not yet ready for prime time.
+	transforms: {
+		from: [
+			{
+				type: 'shortcode',
+				tag: 'contact-form',
+				attributes: {
+					subject: {
+						type: 'string',
+						shortcode: function( named ) {
+							return named.subject;
+						},
+					},
+					to: {
+						type: 'string',
+						shortcode: function( named ) {
+							return named.to;
+						},
+					},
+					submit_button_text: {
+						type: 'string',
+						shortcode: function( named ) {
+							return named.submit_button_text;
+						},
+					},
+				}
+
+			}
+		]
+	},
+	*/
+
+	attributes: {
+		subject: {
+			type: 'string',
+			'default': null
+		},
+		to: {
+			type: 'string',
+			'default': null
+		},
+		submit_button_text: {
+			type: 'string',
+			'default': __( 'Submit »' )
```

Comment:
Would _"Send"_ work as a less technical word here?

Reply:
It's the legacy string, and I'd rather leave it in place so as to not unnecessarily churn translations.